src/Analyzers/Analyzers/src/Microsoft.AspNetCore.Analyzers.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>CSharp Analyzers for ASP.NET Core.</Description>
+    <PackageTags>aspnetcore</PackageTags>
+     <!-- This package is for internal use only. It contains analyzers that are bundled in the .NET Core WebSDK. -->
+    <IsShippingPackage>false</IsShippingPackage>
```

Comment:
These will go in the SDK, so there's no need to produce a package.

Reply:
If there is no need to make a .nupkg file, set `<IsPackable>false</IsPackable>`.